Makefile

```diff
@@ -92,7 +92,7 @@ dockers: $(DOCKERS) docker_ui docker_mqtt
 
 dockers_dev: $(DOCKERS_DEV)
 
-dockers_arm: $(DOCKERS_ARM) docker_ui_arm docker_mqtt_arm
+dockers_arm: docker_arm_mqtt docker_arm_ui $(DOCKERS_ARM) 
```

Comment:
there is a reason for this?

Reply:
And fixing names of variables. Switch back to original order.